tools/docker/env/Dockerfile

```diff
@@ -29,7 +34,9 @@ RUN dpkg --add-architecture i386 && \
 	# These are needed to build Linux kernel:
 	flex bison bc libelf-dev libssl-dev \
 	# qemu-user is required to run alien arch binaries in pkg/cover tests.
-	qemu-user \
+	qemu-user qemu-user-static \
```

Comment:
Can't this work with the existing qemu-user package? I think it should, we already use it to run cross-arch binaries. Installing both and using one or another in various locations looks inconsistent and makes the image larger.

Reply:
Unfortunately no, you need the static one if you build the root image for a target != host.
The reason is that as soon as you enter chroot no host dynamic libs are available (which are used for non-static qemu-s390x) and the dynamic libs inside the chroot are of different architecture. 

Shall i move it to big-env also ?

Reply:
Mkay. Will it work if we replace qemu-user with qemu-user-static then?
qemu-user-static looks like a strictly better superset of qemu-user (but probably larger), so if we install it anyway, we could use it everywhere (?)

Reply:
yes, i think so, but the binaries are of course bigger.

Reply:
sorry, i was wrong, the static one are not the same as the non-static ones !!! you can not boot linux with the static ones. the static ones are for user-space only.